gputop/remote/ajax/metrics.html

```diff
@@ -212,16 +216,12 @@ <h2 id="page-header-metrics">Metrics panel loading</h2>
 
     function draw_filter_checkboxes()
     {
-        if (selected_index < 0)
-            return;
-
```

Comment:
I think I added this recently because I was somehow seeing this code run with selected_index = -1; do we know that that should now never happen?